src/amber/controller/helpers/render.cr

```diff
@@ -2,11 +2,20 @@ module Amber::Controller::Helpers
   module Render
     LAYOUT = "application.slang"
 
-    macro render(template = nil, layout = true, partial = nil, path = "src/views", folder = __FILE__)
+    macro render(template = nil, layout = true, partial = nil, path = "src/views", folder = __FILE__, locals = nil)
       {% if !(template || partial) %}
         raise "Template or partial required!"
       {% end %}
 
+      {{ obj = locals }}
```

Comment:
Please don't remap locals to obj. Just iterate over locals. Macro's are hard enough to read without place holder var names.

Reply:
Done.